src/components/ContentNode.vue

```diff
@@ -100,29 +100,35 @@ function renderNode(createElement, references) {
     ))
   ));
 
-  const renderTableChildren = (rows, headerStyle = TableHeaderStyle.none) => {
+  const renderTableCell = (
+    element, attrs, data, cellIndex, rowIndex, extendedData,
+  ) => {
+    const { colspan, rowspan } = extendedData[`${rowIndex}_${cellIndex}`] || {};
+    // if either is `0`, then its spanned over and should not be rendered
+    if (colspan === 0 || rowspan === 0) return null;
+    return createElement(element, { attrs: { ...attrs, colspan, rowspan } }, (
+      renderChildren(data)
+    ));
+  };
+
+  const renderTableChildren = (rows, headerStyle = TableHeaderStyle.none, extendedData = {}) => {
+    // build the matrix for the array
     switch (headerStyle) {
     // thead with first row and th for each first row cell
     // tbody with rows where first cell in each row is th, others are td
     case TableHeaderStyle.both: {
       const [firstRow, ...otherRows] = rows;
       return [
         createElement('thead', {}, [
-          createElement('tr', {}, firstRow.map(cell => (
-            createElement('th', { attrs: { scope: 'col' } }, (
-              renderChildren(cell)
-            ))
+          createElement('tr', {}, firstRow.map((cell, ci) => (
```

Comment:
```suggestion
          createElement('tr', {}, firstRow.map((cell, cellIndex) => (
```

For me at least, `ci` is pretty confusing to read on its own without the above context from the render function.

Reply:
Yep, makes total sense.